photomaker/pipeline.py

```diff
@@ -344,7 +344,15 @@ def __call__(
         )
         
         # 4. Encode input prompt without the trigger word for delayed conditioning
-        prompt_text_only = prompt.replace(" "+self.trigger_word, "") # sensitive to white space
```

Comment:
These changes are perfect, I like how the way you remove stuff and you add more stuff, If I was a commitment developer I will love to make you a collaborator or whatever it is.

Reply:
Thanks a lot! Glad you liked the changes. Excited to contribute more in the future!